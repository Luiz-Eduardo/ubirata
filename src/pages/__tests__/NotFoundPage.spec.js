import '@testing-library/jest-dom';
import { render } from '@testing-library/vue';
import NotFoundPage from '../NotFoundPage.vue';

const renderComponent = (options) => render(NotFoundPage, {...options });

const route = { fullPath: '/', path: '/' };

describe('NotFoundPage', () => {
  it('renders correctly', () => {
    const { container } = renderComponent();

    expect(container).toMatchSnapshot();
  });
});
