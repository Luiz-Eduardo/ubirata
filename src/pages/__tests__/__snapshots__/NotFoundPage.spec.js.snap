// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFoundPage > renders correctly 1`] = `
<div>
  
  <h1>
    404 Error
  </h1>
  <div
    class="mt-8"
  >
    <p
      class="mb-4"
    >
       I'm sorry, the page you are looking for cannot be found. Please check the URL and try again, or navigate to a different page using the menu above. If you continue to experience issues, please contact our support team for assistance. 
    </p>
  </div>
  
</div>
`;
